DevProject/Packages/manifest.json

```diff
@@ -6,7 +6,7 @@
     "com.unity.ml-agents.extensions": "file:../../com.unity.ml-agents.extensions",
     "com.unity.package-manager-doctools": "1.7.0-preview",
     "com.unity.package-validation-suite": "0.19.0-preview",
-    "com.unity.test-framework": "1.1.22",
+    "com.unity.test-framework": "1.1.24",
```

Comment:
Not strictly necessary in this PR, but rolling forward with new versions here and in the `ProjectSettings/ProjectVersion.txt` files